### PR TITLE
Setup.cfg Update

### PR DESCRIPTION
**Pull request by timmy61109**
If this update is adopted and Flake8, pylint, and pydocstyle are added, the following problems will be avoided.

2021-01-04 18-10-49 的螢幕擷圖

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,21 @@
 [flake8]
-exclude = build,.git,build-pypy,.tox
+exclude =
+    .git,
```
Comment (jelmer) [inline setup.cfg:3]: flake8 doesn't complain about any of these things, why do they need to be added to the exclude list?

File: setup.cfg [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+    .eggs,
+    venv
 
 [mypy]
 ignore_missing_imports = True
+
+[pydocstyle]
+
+[pylint]
+disable = R0205
```
Comment (jelmer) [inline setup.cfg:20]: Please also add pylint to make / GitHub actions, otherwise this will just regress.

**Comment by jelmer** [comment]
(I've converted this back to draft for the moment, please click the "Ready for review" button once it's ready)

File: setup.cfg (hunk 1 above)
Comment (timmy61109) [reply setup.cfg:3]: Eliminating unnecessary inspections can reduce the time and computing resources spent in inspections.

Comment (timmy61109) [reply setup.cfg:3]: These are common places that do not need to be checked, and this project is also applicable

**Comment by jelmer** [comment]
CLosing this for the moment since it's been stalled for > 6 months; feel free to reopen if you'd like to continue working on it.

**Pull request closed without merging**